app/presenters/renewal_letter_presenter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RenewalLetterPresenter < BaseLetterPresenter
+  # Provides the full postal address for the letter.
+  def postal_address_lines
+    lines = [contact_full_name]
+    lines << operator_name unless business_type == WasteExemptionsEngine::Registration::BUSINESS_TYPES[:sole_trader]
```

Comment:
```suggestion
    lines << operator_name unless business_type == sole_trader?
```
The `#sole_trader?` method should be implemented in the engine `CanHaveRegistrationAttributes` to be correct so this is a NIT

Reply:
I agree, but think I'd rather do this for all business types as a separate PR and then update this once the engine change is rolled out. What do you think?

Reply:
I think it makes sense :)